packages/muelu/src/Interface/MueLu_MLParameterListInterpreter_def.hpp

```diff
@@ -84,6 +84,16 @@
 #include "MueLu_NullspaceFactory.hpp"
 #include "MueLu_ParameterListUtils.hpp"
 
+#ifdef HAVE_MUELU_KOKKOS_REFACTOR
+#include "MueLu_CoalesceDropFactory_kokkos.hpp"
+// #include "MueLu_CoarseMapFactory_kokkos.hpp"
+// #include "MueLu_CoordinatesTransferFactory_kokkos.hpp"
+// #include "MueLu_NullspaceFactory_kokkos.hpp"
```

Comment:
How come we don't need NullspaceFactory_kokkos, but the regular NullspaceFactory?